.vib/vib-verify.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
@fmulero Could you review the indentation of this file? 2-spaces indentation.

Reply:
It should be fixed now